.eslintrc

```diff
@@ -12,6 +12,7 @@
     "id-length": 0,
     "new-cap": [2, { "capIsNewExceptions": ["AND"] }],
     "react/jsx-pascal-case": [2, { "allowAllCaps": true }],
+    "import/first": 0,
```

Comment:
it could, but it seems stupid to do it for every file. The reason I did this was because i couldn't figure out how to get Karma to run `setupAdapters.js` before running the test suite, and that is now required.